interview.md

```diff
@@ -11,41 +11,61 @@ _What need is met by your idea? Who are you helping? What is that person's
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+This DSL is aproprite for poker player so that they can input events that happen at the table in real time and allows for more accurate data represention then when they add it to thier notes after the fact. Additionally I keeps all of a players notes, poker data, and suggestions all in one place. The need is for real time note / obervation taking while sitting at the poker table. It would allow the players to take accurate observation in real time but also not take away from their attention to the table. 
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+This idea exites me and other poker players because it combines this notion of a ststitical game with more accurate stastitics as well as provides the player the ability to keep track of past hands without having to recall memory from the last 4 hours of play. I was able to come up with this idea after interveiwing my dad who is a sofware engineer and a poker player. After having a long converstion about things he does at work and things that he does in his free time we narrowed in on poker. Since we both play poker we both found it difficult to keep track of things that happen at the poker table espessally since you can be so focused on the game. Additionally good pker player always write down intersting events that happen at the table such as big hands, how they busted out, and tells of oher players. This idea of a dsl that allows for real time tracking of these events came to life. 
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+real time Language for poker analysis
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+The user would be able to use natural laguage to describe poker hands and events or even small key words/ operations to decribe events at the table. Such as "in my hand I held As(Ace of spades) 2s(2 of spades) I raise from 10$ big blid to 30$ preflop There is one caller, the flop comes ..., the turn comes ..., the oponent in ___ position bet ..., I called ..., the river card was a ..."
+Another possible way would to use short function calls such as:
+newhand(), myHand(), opponet(position on table, what was their action), Ibet(), EndHand() ect... a bunch of poker related functions that store quick and easy info about the table. 
+The goal would be to use a language that allows for quick execution by the user, so that their attention does not devide from the table for more then a few moments. 
+
```

Comment:
I wonder if there's a way to combine the two. function calls seems like too much thinking in terms of remembering all of them, but pure natural language might be hard to parse or end up being more buggy than intended.